docs/src/_guide/orchestration.md

```diff
@@ -26,6 +26,26 @@ meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "tap-gitlab targe
 meltano schedule add daily-gitlab-load --job tap-gitlab-to-target-postgres-with-dbt --interval '@daily'
 ```
 
```

Comment:
@cjohnhanson this text looks good. We'll need an update to https://docs.meltano.com/guide/configuration#specifying-environment-variables as well in the code and precedence blocks as well.

Reply:
@tayloramurphy done.

The docs I added outline the _current_ behavior. Note that it turns out schedule invocation is fairly complex and correcting behavior will need to be downstream of a significant refactor in how we manage environment variables. I've created [this follow-up issue](https://github.com/meltano/meltano/issues/6675) with details.